api/test/programs/test-syscall-in-state/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "test-syscall-in-state"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2021"
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
```

Grumble: `debug` is not used as I see but it can be left for future updates 😄 

```suggestion
gstd = { git = "https://github.com/gear-tech/gear.git" }
```

---

Yep, remove this everywhere